src/isa/riscv64/instr/rvv/vreg.h

```diff
@@ -100,10 +100,12 @@ void longjmp_raise_intr(uint32_t foo);
 #define UNSIGNED     0
 #define SIGNED       1
 
+// set vp dirty
 #define set_mstatus_dirt() \
 do{ \
   if(((mstatus->val >> 9) & 3ull) != 3) {\
     mstatus->val = mstatus->val | (3ull << 9);\
+    mstatus->sd  = 1;\
   } \
 } while (0) \
```

Comment:
Remove dirty macro, and use function instead. fp_set_dirty can be the reference

Reply:
Remove this,  there is a function in vp.c named vp_set_dirty

Reply:
> Remove this, there is a function in vp.c named vp_set_dirty

When I searched the code, I found that vp_set_dirty () was not implemented,